free-programming-books.md

```diff
@@ -1386,9 +1386,9 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Git Reference](https://web.archive.org/web/20170602211147/http://gitref.org/)
 * [Git Succinctly, Syncfusion](https://www.syncfusion.com/resources/techportal/ebooks/git) (PDF, Kindle) (email address *requested*, not required)
 * [Git Workflows](http://documentup.com/skwp/git-workflows-book)
-* [How to Collaborate on Github](https://github.com/eonist/How-to-collaborate-on-github) - André J
-* [Intoduction to Git and Github](https://launchschool.com/books/git) - Launch School
-* [Intoduction to Git and Github - Tutorial](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
+* [How to Collaborate on GitHub](https://github.com/eonist/How-to-collaborate-on-github) - André J
+* [Intoduction to Git and GitHub](https://launchschool.com/books/git) - Launch School
+* [Intoduction to Git and GitHub - Tutorial](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
```

Comment:
It seems to be the title of this resource doesn't even contain GitHub, it's only "Introduction to Git", could we update it?

```suggestion
* [Intoduction to Git](http://cse.unl.edu/~cbourke/gitTutorial.pdf) - Dr. Chris Bourke (PDF)
```

Reply:
Thanks! And the mis-spelling "Int**r**oduction" also fixed.

Reply:
Didn't even notice that! Nice catch! ^-^'